mem/mem_bsd.go

```diff
@@ -0,0 +1,91 @@
+// +build freebsd openbsd
+
+package mem
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+const swapCommand = "swapctl"
+
+// swapctl column indexes
+const (
+	nameCol     = 0
+	totalKiBCol = 1
+	usedKiBCol  = 2
+)
+
+func SwapDevices() ([]*SwapDevice, error) {
+	return SwapDevicesWithContext(context.Background())
+}
+
+func SwapDevicesWithContext(ctx context.Context) ([]*SwapDevice, error) {
+	swapCommandPath, err := exec.LookPath(swapCommand)
+	if err != nil {
+		return nil, fmt.Errorf("could not find command %q: %w", swapCommand, err)
+	}
+	output, err := exec.Command(swapCommandPath, "-lk").Output()
```

Comment:
sorry, I forget to mention but could you change to use `invoke.CommandWithContext` [like here](https://github.com/shirou/gopsutil/blob/master/mem/mem_solaris.go#L105) please? This invoker can fake command output

Reply:
Apologies for the long delay in responding to this! I've addressed this feedback now.